components/core/src/package/install.rs

```diff
@@ -541,14 +561,15 @@ impl PackageInstall {
         Ok(env)
     }
 
-    /// Return the parsed contents of the package's `RUNTIME_ENVIRONMENT` metafile as a `HashMap`,
+    /// Return the parsed contents of the package's environment` metafile as a `HashMap`,
     /// or an empty `HashMap` if not found.
     ///
-    /// If no value of `RUNTIME_ENVIRONMENT` is found, return an empty `HashMap`.
-    fn runtime_environment(&self) -> Result<HashMap<String, String>> {
-        match self.read_metafile(MetaFile::RuntimeEnvironment) {
+    /// If no value of file is found, return an empty `HashMap`.
+    fn runtime_environment_file_map(&self, metafile: MetaFile) -> Result<HashMap<String, String>> {
+        match self.read_metafile(metafile) {
             Ok(ref body) => Self::parse_runtime_environment_metafile(body),
-            Err(Error::MetaFileNotFound(MetaFile::RuntimeEnvironment)) => Ok(HashMap::new()),
+            Err(Error::MetaFileNotFound(MetaFile::RuntimeEnvironment))
+            | Err(Error::MetaFileNotFound(MetaFile::RuntimeEnvironmentPaths)) => Ok(HashMap::new()),
```

Comment:
Could this just be `Err(Error::MetaFileNotFound(_)) => Ok(HashMap::new())` instead of checking for the specific file types?